### DISCUSSION TITLE
Media for the metaverse festival / metaverse burn

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/media#65] [General]
This discussion can hold the work for the continuing media related things for our festival. 
We need to make 
1. A webpage saying in broad strokes what we're doing and where people can come check it out, and then
2. A press release to send to zines or whatever and forums to start spreading the word and generate excitement. (I am thinking more "traditional" online media outlets. My personal interest is in the non-techie world, of course we need where the virtual world people hang out but that is not my strength.)

This page is a start to draw content from: https://hackmd.io/@indiebio/metaverse-float-parade

We need to update it with the content jin has created so far, and add content how-to's to get involved as they become clear. 
for example:
https://sketchfab.com/3d-models/hyperfy-simple-cars-parade-float-d1a2b04bc8684e31a2c59923630ec516
https://hyperfy.io/cars

**Comment by madjin** [comment]
Down to work on this sometime ad-hoc, will hash out some stuff we can use for flyer

**Comment by jimmy6DOF** [comment]
I think a list of potential collaborating xyz-Verses could help extend the reach if we can set up the same portal network used in the metaverse-traversal efforts if we can nail down a list of the active sites we can try to get them on board.  Closest I can find is this form the pre AWE program ...... https://github.com/omigroup/omigroup/issues/389

**Comment by indiebio** [comment]
General comments from the meeting (15 Aug): Events are well within the capability of OMI group and works well for networking with other metaversal groups.
